docs/docs/use-the-factory-orchestrator-app.md

```diff
@@ -30,3 +30,50 @@ A 'Re-run' button will also appear next to a Task if the TaskList is done execut
 If you click on a Task, the results page will load and show you the status of the latest "run" (TaskRun) of that Task, including the any output of the Task. The results page also allows you to see the log file path for that run. You can also use the buttons at the top of the page to view older or newer runs of the Task, provided it has been run multiple times.
 
 ![Clicking on a Task that has run](./images/test-results.png)
+
+# Factory Orchestrator utilities
+
+In addition to providing a graphical interface to create, manage, and run Tasks and TaskLists. The Factory Orchestrator app also includes some basic utilities intended as a starting point for integration into a manufacturing line, fault analysis workflow or developer inner loop.
+
+<!-- ## UWP Apps
+
+This launches a UWP app that's installed on a device under test (DUT). This allows you to launch a UWP directly from the Factory Orchestrator app by clicking on its name in the list of installed UWP apps.
+
+Your device must be configured to launch into an environment that supports launching UWP apps.
+
+You can exit a launched UWP with ALT+F4, or from Windows Device Portal. -->
+
+## Command Prompt
+
+A basic, non-interactive, command prompt that allows you to troubleshoot without having use other methods like SSH or Windows Device Portal to connect to your DUT.
+
+While you can run commands and see output when using the built-in command prompt in Factory Orchestrator, it's not an interactive shell. If you run a command that requires additional input, you won't be able to enter the additional input.
+
+![The Command Prompt screen](./images/fo-cmd.png)
+
+## File Transfer
+![The Transfer Files screen](./images/fo-transfer-files.png)

```

Comment:
this photo doesn't currently render in the preview.

Reply:
https://spaceisfun.github.io/FactoryOrchestrator/use-the-factory-orchestrator-app/ its visible here so i think im good?

Reply:
PR rich preview seems to expect it checked in to microsoft/FO main branch so yeah thats not going to render 😝